api/envoy/config/bootstrap/v3alpha/bootstrap.proto

```diff
@@ -151,6 +151,11 @@ message Bootstrap {
   // <server_statistics>` if specified. Envoy will not process this value, it will be sent as is to
   // :ref:`stats sinks <envoy_api_msg_config.metrics.v3alpha.StatsSink>`.
   google.protobuf.UInt64Value stats_server_version_override = 19;
+
+  // Always use TCP queries instead of UDP queries for DNS lookups.
+  // This may be overridden on a per-cluster basis in cds_config,
+  // when custom DNS resolvers are used and use_tcp_for_dns_lookups is specified.
```

Comment:
s/when custom DNS resolvers .../ when `dns_resolvers` and `use_tcp_for_dns_lookups` are specified